src/screens/VolumeCalibrationScreen.tsx

```diff
@@ -0,0 +1,231 @@
+                  testCurrentVolume()
+                }
+              },
+            },
+            {
+              text: 'Not Sure',
+              onPress: () => {
+                // Restart countdown
+                testCurrentVolume()
+              },
+            },
+          ],
+          { cancelable: false },
+        )
+      })
+    }
+
+    // Run volume test
+    testCurrentVolume()
+  }
+
+  const validateRating = () => {
+    // Respond if no rating selected
+    if (volumeRating === undefined) {
+      Alert.alert(
+        'Rate the volume',
+        'Please use the rating scale to describe how you felt about the played sound',
+      )
+    }
+
+    // User finds volume painful, let's verify they want to continue
+    else if (volumeRating === 9) {
+      Alert.alert(
+        'Are you sure',
+        'You may hear sounds at this volume multiple times during the experiment. Do you think you can tolerate this volume?',
+        [
+          {
+            text: 'Yes',
+            onPress: () => onFinishCalibration(volume.current),
+          },
+          {
+            text: 'No',
+            onPress: () => setStage(3),
+          },
+        ],
+        { cancelable: false },
+      )
+    }
+
+    // See if rating is beyond threshold
+    else if (volumeRating > 5) {
+      onFinishCalibration(volume.current)
+    }
+  }
+
+  const resetCalibration = () => {
+    volume.current = 0.5
+    setVolumeRating(undefined)
+    startCalibration()
+  }
+
+  return (
+    <Box
+      flex={1}
+      alignItems="center"
+      pt={24}
+      px={8}
+      backgroundColor="greenPrimary"
+    >
+      <Entypo name="sound" size={100} color={palette.darkGrey} />
+
+      <Text mt={8} variant="caption" textAlign="center">
+        For this experiment it is important that the volume of the sound is
+        uncomfortable but not painful.
+      </Text>
+
+      {/* Instruction Stage */}
+      {stage == 0 && (
+        <Box
+          flex={1}
+          mt={15}
+          pb={14}
+          justifyContent="flex-end"
+          alignItems="center"
+        >
+          <Text pb={10} variant="caption2" textAlign="center">
+            Once you click the button below you will be asked to rate the volume
+            played through your headphones.
+          </Text>
+          <Button
+            variant="primary"
+            label="Start"
+            alignSelf="flex-end"
+            onPress={startCalibration}
+          />
+        </Box>
+      )}
+
+      {/* Stimuli Countdown */}
+      {stage == 1 && countdown > 0 && (
+        <Text variant="heading" fontSize={100} mt={24}>
+          {countdown}
+        </Text>
+      )}
+
+      {/* Volume Rating Scale */}
+      {stage == 2 && (
+        <Box
+          flex={1}
+          mt={15}
+          pb={14}
+          justifyContent="flex-end"
+          alignItems="center"
+        >
+          <TrialRatingScale
```

Comment:
I suppose now that this is reused here, it's no longer just a `TrialRatingScale` 😛